src/Settings.ts

```diff
@@ -78,12 +78,15 @@ module Settings
     }
 
     export function getUrlVars()
-    {
+	{
+		//TODO
+		//VERIFICAR O RETORNO QUE PODE ESTAR INCORRETO PARA O METODO
```

Comment:
?

Reply:
I'm brazilian, that's why the spelling. 
The changes initaly was for me to test.
I let that "TODO" to check if the return "" would still make the method work.
I forgot to remove it